packages/boxel-ui/addon/bin/rebuild-icons.mjs

```diff
@@ -34,6 +34,7 @@ let componentsToGenerate = fs
 componentsToGenerate.sort((a, b) => a.name.localeCompare(b.name));
 
 for (const c of componentsToGenerate) {
+  console.log(`Generating ${c.sourceFile}...`);
```

Comment:
I added this because I was seeing a failure when rebuilding the icons and seeing the file it stopped on helped me understand the problem.